frontend/app/views/fragments/pricing/billingPeriodChoice.scala.html

```diff
@@ -47,6 +47,9 @@
             generateChargeAmount: function() {
                 return formattedDiscountedCharge(checkoutForm.currency, this.amount, this.discount);
             },
+            generateDisplayAmount: function() {
```

Comment:
Yeah we should remove them from the choices array too. It was there to make the template cleaner, so it doesn't have to call back up the object model to find the correct generateDisplayAmount function, it's there on the choice it's looped over.